batch/scripts/run_pixetl.sh

```diff
@@ -35,4 +35,7 @@ fi
 ARG_ARRAY+=("${JSON}")
 
 # Run pixetl with the array of arguments
-pixetl "${ARG_ARRAY[@]}"
\ No newline at end of file
+pixetl "${ARG_ARRAY[@]}"
+
+# Use this command instead when debugging memory issues
+#mprof run -M -C -T 1 --python /usr/local/app/gfw_pixetl/pixetl.py "${ARG_ARRAY[@]}"
```

Comment:
Do we want to keep this in production code, I feel like this should be something for the developper documentation

Reply:
I left it in as a comment because I didn't want to have to figure it out again, but you're right, the README would be a better place for it.